.storybook/custom/custom.scss

```diff
@@ -308,3 +308,28 @@ body.sb-show-main {
   color: #32363a;
   color: var(--sapTextColor, #32363a);
 }
+
+.fdsb-layout-panel-playground {
+  border: 1px solid navy;
+  border-radius: 4px;
+  padding: 10px;
+  margin: 0;
+  background: gold;
+  font-size: 14px;
+  box-shadow: 0 5px 20px 0 blue;
+}
+
+.fddocs.fd-container {
+  min-height: 20px;
+  padding: 20px;
+  background: lightblue;
+  border: 1px solid yellow;
```

Comment:
Why yellow 😆

Reply:
oh I see it's for the examples - it feels weird to use a class like this - why not a docs-container?